tests/integration/any_charm.py

```diff
@@ -27,21 +27,19 @@ def __init__(self, *args, **kwargs):
             kwargs: Variable list of positional keyword arguments passed to the parent constructor.
         """
         super().__init__(*args, **kwargs)
-        self.ingress = IngressRequires(
-            self,
-            {"service-hostname": self.app.name, "service-name": self.app.name, "service-port": 80},
-        )
+        require_nginx_route(charm=self, service_hostname=self.app.name, service_name=self.app.name, service_port=80)
 
-    def update_ingress(self, ingress_config):
-        """Update Ingress config.
+    def delete_nginx_route_relation_data(self, field: str) -> None:
+        """Delete one data filed from the nginx-route relation data.
```

Comment:
I don't understand the comment as is, can you elaborate?

Reply:
Or is it a typo - s/filed/field/ ?

Reply:
It is a typo, fixed :+1: